src/agent.cc

```diff
@@ -3,6 +3,12 @@
 
 namespace tagt::xchg::agent {
 
+/**
+ * Prints out XCHG=> followed by whatever string data is associated with agent ID (uint_16)

```

Comment:
I think this comment is one of those comments that talks too much about implementation rather than purpose. Right now this is temporary behavior and in the future agent::reply is going to send a message to the agent over TCP, so I think the documentation should reflect that. Something like:
```
/**
 * Sends a reply to a client (agent)
 * @param agent The ID for the client to contact
 * @param response The packet to send to the client
```